components/brave_rewards/resources/ui/components/checkout/addFundsPanel/index.tsx

```diff
@@ -165,7 +167,11 @@ export function AddFundsPanel (props: AddFundsPanelProps) {
         />
       </PurchaseButtonRow>
       <TermsOfSale>
-        <span dangerouslySetInnerHTML={{ __html: locale.get('addFundsTermsOfSale') }} />
+        <span>
+          {tags.beforeTag}
+          <a href='javascript:void 0'>{tags.duringTag}</a>
+          {tags.afterTag}
+        </span>
```

Comment:
TEST PLAN NEEDED - i couldn't trigger add funds because i didn't have 25 BAT

Reply:
All component files under `ui/components/checkout` are not currently used in the browser build, so there's no test plan at this time.